src/item.cpp

```diff
@@ -2250,7 +2250,7 @@ std::string item::info( std::vector<iteminfo> &info, const iteminfo_query *parts
 
             insert_separation_line();
             info.push_back( iteminfo( "DESCRIPTION",
-                                      string_format( _( "Disassembling this item takes %s and might yield: %s." ),
+                                      string_format( _( "Disassembly takes %s and might yield: %s." ),
```

Comment:
I think the original here is better. Does the code support the ability to embed the string for the item name so it reads
"Disassembling the %n takes %s and might yield: %s." ? A couple times i've fat fingered and disassembled something I didn't mean to.